build/images/base/Dockerfile

```
@@ -13,6 +13,27 @@
 # limitations under the License.
 
 ARG BUILD_TAG
+
+FROM ubuntu:22.04 AS bgp
```

```suggestion
FROM ubuntu:22.04 AS gobgp-binary
```

for accuracy and consistency with `cni-binaries`